web-ui/src/main/resources/catalog/components/common/ows/OWSService.js

```diff
@@ -390,59 +460,69 @@
             return defer.promise;
           },
 
-          getLayerExtentFromGetCap: function(map, getCapLayer) {
+          getWmsLayerExtentFromGetCap: function(map, getCapLayer, isCustomDef) {
```

Comment:
Since this function is now being used for WMS layers exclusively, its name should reflect that.
Also added the `isCustomDef` parameter.